### PR TITLE
ci: allow to disable boost::stacktrace

### PR DESCRIPTION
**Pull request by sileht**

### CODE REVIEW

File: src/oatppjsonapi.cc [hunk 1]
```diff
@@ -268,12 +270,15 @@ namespace dd
       }
   }
 
+#if USE_BOOST_BACKTRACE
   void OatppJsonAPI::abort(int signum)
   {
+    (void)signum;
```
Comment (fantes) [inline src/oatppjsonapi.cc:276]: (void) signum not needed ?

**Review by fantes — APPROVED** [review]

**Review by beniz — APPROVED** [review]

**Comment by beniz** [comment]
@Mergifyio update

**Comment by beniz** [comment]
@Mergifyio update

**Comment by sileht** [comment]
@Mergifyio update

**Pull request merged**